docs/documentation/authorization_services/topics/getting-started-overview.adoc

```diff
@@ -7,4 +7,4 @@ get started with {project_name} Authorization Services:
 * {quickstartRepo_link}/tree/latest/jakarta/servlet-authz-client[Securing a JakartaEE Application in Wildfly]
 * {quickstartRepo_link}/tree/latest/spring/rest-authz-resource-server[Securing a Spring Boot Application]
 * link:https://quarkus.io/guides/security-keycloak-authorization[Securing Quarkus Applications]
-* {adapterguide_link_nodejs_adapter}[Securing Node.js Applications]
\ No newline at end of file
+* *Keycloak Node.js adapter* in the link:https://www.keycloak.org/guides#securing-apps[securing apps] section
```

Comment:
Same like my other comment (can be hopefully attribute instead of hardcoded link).